<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="file_name" value="$(find pointmap)/map/pointcloudmap_3831137000.pcd" />
  <arg name="rvizconfig" default="$(find point_cloud_map_server)/rviz/point_cloud_map_server.rviz" />
  
  <!-- point_cloud_map_server -->
  <node pkg="point_cloud_map_server" type="map_loader" name="point_cloud_map_loader" args="$(arg file_name)" output="screen">
    <param name="frame_id" value="map"/>
    <param name="width" value="384"/>
    <param name="height" value="384"/>
    <param name="resolution" value="0.05"/>
    <param name="origin_x" value="-10.0"/>
    <param name="origin_y" value="-10.0"/>
  </node>

  <!-- rviz -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />
</launch>
